ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json

```diff
@@ -839,6 +839,7 @@
   "home": {
     "paragraph1Header": "Have a direct impact on the future of leiomyosarcoma.",
     "paragraph1Text": "If you have ever been diagnosed with leiomyosarcoma, join a nationwide movement of patients to contribute your samples, your medical records, and your voice to leiomyosarcoma research. Together, we can speed the development of future therapies.",
+    "fundingText": "Count Me In’s Osteosarcoma and Leiomyosarcoma Projects are made possible by federal funding through the Participant Engagement and Cancer Genome Sequencing (PE-CGS) grant as part of the Cancer Moonshot Initiative. Learn more about PE-CGS <a class=\"hereLink\" href=\"https://pe-cgs.org/\" target=\"_blank\"><u>here.</u></a>",
```

Comment:
wondered if both pec-cgs study names should be there in the text OR only lms

Reply:
I think it's both, but check with Elana.

Reply:
Since spanish (OS only has Spanish) tex also had both, I believe its both for sure